addons/web_editor/static/src/js/wysiwyg/wysiwyg.js

```diff
@@ -3412,6 +3412,19 @@ export class Wysiwyg extends Component {
             delete el.dataset.bgSrc;
         } else {
             el.setAttribute('src', newAttachmentSrc);
+            // Also update carousel thumbnail.
+            const carouselEl = el.closest(".carousel");
+            if (carouselEl) {
+                const carouselInnerEl = el.closest(".carousel-inner");
+                const carouselItemEl = el.closest(".carousel-item");
+                if (carouselInnerEl && carouselItemEl) {
```

Comment:
not sure to understand why you need to do this

Reply:
Because it felt like `.carousel` was a bit too easy to match, and I don't want the next line to fail if those elements were not in there.